resources/migrations/000_migrations.yaml

```diff
@@ -14312,7 +14312,6 @@ databaseChangeLog:
       id: v47.00-001
       author: calherries
       comment: Added 0.47.0 -- set base-type to type/JSON for JSON database-types for postgres and mysql
-      validCheckSum: ANY
```

Comment:
why remove this?

Reply:
`validCheckSum: ANY` wasn't supposed to be there. I used it in development to edit the SQL. And removing `validCheckSum` doesn't change the checksum, no. I just tested by removing validCheckSum from a migration and starting the server again. No issues.